style.css

```diff
@@ -439,3 +466,199 @@ body {
   border: none;
   border-radius: 10px;
 }
+
+.containerDesktopUl {
+  display: flex;
+  justify-content: center;
+}
+
+.unlistbar {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width: 17rem;
+  padding: 0;
+}
+
+/*
+tablet
+*/
+@media (min-width: 768px) {
+  .container,
+  header {
+    display: none;
+  }
+
+  .containerDesk {
+    display: flex;
+    justify-content: center;
+    align-items: baseline;
+  }
+
+  .tittleContainer {
+    margin-top: 0%;
+    font-family: 'Inter', sans-serif;
+    background-image: url(/imgdesktopversion/Header-llustration-desktop.svg);
+  }
+
+  .imgOneMovil {
+    display: none;
+  }
+
+  .welcome {
+    padding: 0 20px;
+  }
+
+  .tittleContainer p {
+    padding: 0 70px;
+    margin-top: 0;
+  }
+
+  .mediaContainer {
+    width: 29%;
+  }
+
+  .cardGrid {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .cardGridOne {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .skillsItem {
+    margin: 22px;
+    display: grid;
+    gap: 24px;
+    justify-items: center;
+  }
+
+  .aboutMeCard {
+    grid-template-columns: repeat(3, 1fr);
+    padding-bottom: 11px;
+    margin-bottom: 5%;
+    border: solid 1px #d0d9d4;
+    background-color: #ebf0ee;
+    border-radius: 10px 10px 10px 10px;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 100%;
+  }
+
+  .containerThirdTittleForm {
+    margin: 24px;
+  }
+
+  .formThirdTittle {
```

Comment:
- [ ] Good job 👏. However, I have some suggestions to help you improve your project, the layout of your `formThirdTitle` section is a bit different compared to the Figma design. 

To help you fix it add `margin: 0 140px;` to your  `formThirdTittle ` class; And update the `margin` of your `ThirdTitle` to `margin: 5% 5% 5% 10%;`.